src/main/kotlin/br/com/zup/op/events/infra/annotation/PayloadValidator.kt

```diff
@@ -0,0 +1,17 @@
+package br.com.zup.op.events.infra.annotation
```

Comment:
remove this package `infra.annotation` and move this class to `infra.validation` package